packages/x-components/src/x-modules/queries-preview/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './events.types';
+export * from './store';
+export * from './x-module';
+export * from './wiring';
```

Comment:
Include and export of the `queries-preview` module in `packages/x-components/src/index.ts` too.

Reply:
Done!